osmosis-1/osmosis.zone_assets.json

```diff
@@ -2252,6 +2252,11 @@
           "withdraw_url": "https://portalbridge.com/cosmos/"
         }
       ]
+    },
+    {
+      "chain_name": "osmosis",
+      "base_denom": "factory/osmo1mlng7pz4pnyxtpq0akfwall37czyk9lukaucsrn30ameplhhshtqdvfm5c/ulvn",
+      "osmosis_verified": false
```

Comment:
"osmosis_verified" will immediately be true, although for now we need "osmosis_unlisted": "true"